### DISCUSSION TITLE
After deleting a data series, the UI deletion cannot be called.

### DISCUSSION THREAD

**Discussion by SailRuo** [jfree/jfreechart#395] [Q&A]
Children: duplicate children added: parent = ListViewSkin$2@4e4efb64[styleClass=cell indexed-cell list-cell]'null'

Button delete = FromsUtil.getButtonNoborder("", ImageFactory.getImageView(ImageFactory.delete_svg), 32.0, 32.0);
            Button deleteUI = FromsUtil.getButtonNoborder("", ImageFactory.getImageView(ImageFactory.delete_svg), 32.0, 32.0);
            delete.setOnAction(event -> {
                dataset.setNotify(false);
                for (XYSeries series : seriesList) {
                    dataset.removeSeries(dataset.getSeriesIndex(series.getKey()));
                }
                dataset.setNotify(true);
            });
            deleteUI.setOnAction(event -> {
                Platform.runLater(() -> {
                    layerListView.getItems().remove(this);
                });
            });

 This is my test code. After deleting the data in the first step, delete the data in the layer corresponding to the UI and keep reporting errors
![image](https://github.com/jfree/jfreechart/assets/52062893/e992ef45-70a2-4a4a-afb7-6966e00c3daa)

**Comment by trashgod** [comment]
Be sure you are using recent versions of [jfreechart-fx](https://github.com/jfree/jfreechart-fx) and dependencies. The message `duplicate children added` suggests a JavaFX problem; in particular, "A [`Node`](https://openjfx.io/javadoc/21/javafx.graphics/javafx/scene/Node.html) may occur _at most_ once anywhere in the scene graph."

**Reply by SailRuo** [reply]
Thank you for your reply, the problem has been solved, when javafx vbox remove child node, because I use lombok @Data annotation, it overrides my hashcode method, so remove fails!

**Reply by SailRuo** [reply]
![image](https://github.com/jfree/jfreechart/assets/52062893/bda0b545-8a88-447e-9452-d8be95d2e7f6)
